tls/s2n_kem_preferences.h

```diff
@@ -19,8 +19,13 @@
 #include "tls/s2n_kex.h"
 
 struct s2n_kem_preferences {
+    /* kems used for hybrid TLS 1.2 */
     uint8_t kem_count;
     const struct s2n_kem **kems;
+
+    /* kem_groups used for hybrid TLS 1.3 */
+    uint8_t kem_group_count;
+    const struct s2n_kem_group **kem_groups;
```

Comment:
I think we should add `tls13` as part of the name.

Reply:
Done